src/assets/scripts/client/utilities/radioUtilities.js

```diff
@@ -3,6 +3,7 @@ import _compact from 'lodash/compact';
 import _map from 'lodash/map';
 import { round } from '../math/core';
 import { tau, radians_normalize } from '../math/circle';
+import AirportController from '../airport/AirportController';
```

Comment:
Github is complaining a circular dependency here. We are going to need to figure out a way around that, if we can.